burnman/output_seismo.py

```diff
@@ -0,0 +1,267 @@
+# This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
+# Copyright (C) 2012 - 2015 by the BurnMan team, released under the GNU
+# GPL v2 or later.
+
+
+from __future__ import absolute_import
+
+import numpy as np
+import warnings
+import scipy.integrate
+import matplotlib.pyplot as plt
+
+from . import tools
+from . import constants
+from . import seismic
+from . import geotherm
+
+def write_axisem_input(rock, min_depth=670.e3, max_depth=2890.e3, T0= 1900,name='burnmantestrock',
+                       axisem_ref='axisem_prem_ani_noocean', plotting=False):
+    """
+    Writing velocities and densities to AXISEM (www.axisem.info) input file
```

Comment:
Can you please document the arguments? Is ``burnmantestrock`` a good default? I would prefer if it was the full filename that is given here (including extension).